go.mod

```diff
@@ -84,8 +85,6 @@ replace (
 	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
 	github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
 	github.com/wercker/stern => github.com/oam-dev/stern v1.13.0-alpha
-	// fix build issue https://github.com/ory/dockertest/issues/208
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
```

Comment:
I remove this replacement because the [issue ](https://github.com/ory/dockertest/issues/208) seems fixed already. And honnef.co/go/tools depends on newer version of golang.org/x/sys. 
@zzxwill  Could you please help to verify this fix would not bring breaking issues?

Reply:
Go head:)